oaknational/Oak-Web-Application
# Change swimming text to include "water safety"

## Description
Change "Swimming" text to be "Swimming and water safety"

## Issue(s)
Fixes

 - `CUR-954`
 - `CUR-955`
 - `CUR-956`
 - `CUR-957`
 - `CUR-958`


## How to test

 - Enable cycle 2 locally and test for copy change
 - Go to https://deploy-preview-2867--oak-web-application.netlify.thenational.academy check for regressions
